jbpm-container-test/integration-deps/issue-keeper/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>integration-deps</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>idep-issue-keeper</artifactId>
+
+  <name>Integration Dependencies :: Issue Keeper</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>link.bek.tools</groupId>
```

Comment:
Is there any benefit of using Issue Keeper in upstream? All tests should pass and if not, they must be fixed.